manpages/lumina-calculator.8

```diff
@@ -0,0 +1,21 @@
+.Dd October 24, 2017
+.Dt LUMINA-CALCULATOR 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-calculator
+.Nd is a full scientific calculator with a simplistic interface.
+
+.Sh SYNOPSIS
+.Nm
+
+.Sh DESCRIPTION
+.Nm
+is a simple scientific calculator interface. It has exponential, trigonometric, and
+logarithmic functions. It can also calcualte percentages, square roots, and hyperbolic
+trigonometric functions. While using the utility it keeps a history of previous calculations
+and the history can be saved as a txt file on the system.
```

Comment:
-> The utility keeps a history of previous calculations. This history can be saved as a .txt file.